shell/platform/windows/flutter_windows_view.h

```diff
@@ -46,8 +46,23 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate {
   // Get the view's unique identifier.
   FlutterViewId view_id() const;
 
-  // Creates rendering surface for Flutter engine to draw into.
-  // Should be called before calling FlutterEngineRun using this view.
+  // Whether this view is the implicit view.
+  //
+  // The implicit view is a special view for backwards compatibility.
+  // The engine assumes it can always render to this view, even if the app has
+  // destroyed the window for this view. The embedder must ignore presents to
+  // this view after it is destroyed.
+  //
+  // The implicit view is the only view that can be created before the
+  // engine is launched.
```

Comment:
```suggestion
  // Whether this view is the implicit view.
  //
  // The implicit view is the first view created on an engine, and has some special properties
  // in order to keep backwards compatibility.
  // * The implicit view can be created before the engine is launched.
  // * Even after the implicit view is removed, the Dart app still thinks the implicit view exists
  //    and can render into the view (with no effects).
```

Reply:
Is there something you think is unclear or missing in the original comment? I currently prefer the original comment, but I'm happy to expand/reword anything in it.

Reply:
Yeah.
* I think it is clearer to first tell the reader the criterion that makes a view considered an implicit view, then lists the differences of an implicit view that interest the reader.
* The original version only says that the implicit view special, not how. With the details hidden, the mechanism feels uncontrollable and scary.
* The doc is for the end user, so "The embedder must ignore presents to this view after it is destroyed" is not useful here.
* The original version did not mention the 2nd item in my version, which I think is as important as the 3rd version, since both describes the discrepancy between the embedder views and the Dart views and can potentially confuse the developer.
* The sentence "The implicit view is the only view that can be created before the engine is launched." is just one of the many differences and doesn't have to be in a separate paragraph.

Reply:
Nevermind, I just realized that this entire file is not for the end user but is internal.

Reply:
Still, I think we should still add:
* The implicit view is the first view created on the engine.
* The engine assumes it can always render to this view, even **before the app creates the `FlutterWindowsView`** or after the app has destroyed the window for this view.

Reply:
Thanks for expanding. This doc is not for end users, this doc is for Windows embedder developers. This method will never be called by an end user.

I'll clarify that the embedder must ignore presents to this view before it is created.

Reply:
Updated!